python/dgl/data/chem/csv_dataset.py

```diff
@@ -11,18 +11,51 @@
 from ..utils import download, get_download_dir, _get_dgl_url, Subset
 
 class CSVDataset(object):
+    """CSVDataset
+
+    This is a general class for loading data from csv or pd.DataFrame.
+
+    In data pre-processing, we set non-existing labels to be 0, and returning mask with 1 where label exists.
+
+    All molecules are converted into DGLGraphs. After the first-time construction, the
+    DGLGraphs will be saved for reloading so that we do not need to reconstruct them every time.
+
+    Parameters
+    ----------
+    df: pandas.DataFrame
+    Dataframe including smiles and labels. Can be loaded by pandas.read_csv(file_path).
+    One column includes smiles and other columns for binary labels.
```

Comment:
I suppose this dataset does not necessarily need to be for binary classification?